solvers/milp_LP_HL/try_script.py

```diff
@@ -0,0 +1,26 @@
+from core import Instance, Experiment
```

Comment:
this file should be in the directory `execution`. Or better: just delete it since we already have a file in that directory that does the same thing.